dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/UserUploadsControllerTest.kt

```diff
@@ -26,10 +25,4 @@ class UserUploadsControllerTest {
         }
         assertEquals("Client error : 403 ", exception.message)
     }
-
```

Comment:
Warum ist der gelöscht?

Reply:
The test description is false. it checks that no dataset has been uploaded by dataland reviewer.

with the new logic, we are indeed uploading a dataset as a reviewer. So the list should not be empty anymore

Reply:
But the technical User "Reviewer" does not upload data, correct? Other users who have the keycloak reviewer role in addtion to other roles like the technical User "Admin" upload data. But the "Reviewer" user should still have no uploads.

Reply:
ok